demos/TicTacToe/Classes/MainMenuScene.h

```diff
@@ -0,0 +1,19 @@
+#ifndef TICTACTOE_DEMO_CLASSES_MAINMENU_SCENE_H_
+#define TICTACTOE_DEMO_CLASSES_MAINMENU_SCENE_H_
+
+#include "cocos2d.h"
+#include "ui\CocosGUI.h"
```

Comment:
Use forward slashes. They're necessary on *nix systems, and compatible with windows even though they're not the default directory separator.